spec/unit/provider/service/macosx_spec.rb

```diff
@@ -60,14 +60,14 @@
 
     ["Daemon", "Agent"].each do |service_type|
       ["redis-server", "io.redis.redis-server"].each do |service_name|
-        ["10.9", "10.10"].each do |platform_version|
+        ["10.9", "10.10", "10.11"].each do |platform_version|
           let(:plist) {'/Library/LaunchDaemons/io.redis.redis-server.plist'}
           let(:session) { StringIO.new }
           if service_type == 'Agent'
             let(:plist) {'/Library/LaunchAgents/io.redis.redis-server.plist'}
             let(:session) {'-S Aqua '}
             let(:su_cmd) {'su igor -c'}
-            if platform_version != "10.10"
+            if not ["10.10", "10.11"].include?(platform_version)
```

Comment:
since this seems to be < 10.10 vs >= 10.10, should the code be more like:

```
let(:su_cmd) {'su -l igor -c'}
if platform == 10.9
  let(:su_cmd) {'su igor -c'}
end
```

That way we don't have to update it for every future version?